2023-09-12 17:35:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "4" for parameter "parameterIndex" [90008-206]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:665)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:302)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1395)
	at org.h2.jdbc.JdbcPreparedStatement.setString(JdbcPreparedStatement.java:378)
	at model.VetDAO.create(VetDAO.java:37)
	at model.Main.main(Main.java:12)
2023-09-12 17:38:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "4" for parameter "parameterIndex" [90008-206]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:665)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:302)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1395)
	at org.h2.jdbc.JdbcPreparedStatement.setString(JdbcPreparedStatement.java:378)
	at model.VetDAO.create(VetDAO.java:37)
	at model.Main.main(Main.java:12)
